src/oc/fileio.cpp

```diff
@@ -947,4 +947,22 @@ int Fprintf(FILE* stream, const char *fmt, ...) {
   return n;
 }
 
+void hoc_execerr_ext(const char* fmt, ...) {
```

Comment:
one line doctoring would be nice

Reply:
Sorry. I don't understand what would be nice.

Reply:
I think pramodk means that adding a docstring to this function would be nice, even if it's just one line.

Reply:
Can't find the right emoji for that one.